odoo/addons/test_orm/tests/test_fields.py

```diff
@@ -1931,21 +1931,23 @@ def test_32_prefetch_missing_error(self):
         existing.categories
 
         # invalidate 'categories' for the assertQueryCount
+        self.env.transaction.clear_access_cache()
         records.invalidate_model(['categories'])
-        with self.assertQueryCount(4):
+        with self.assertQueryCount(5):
```

Comment:
For the record, this adds a query, but this use-case is about prefetching non-existing records. It is not representative of regular code.